final_prep/annotations.rb

```diff
@@ -3,41 +3,63 @@
 # Use the # to create a new comment
 
 # Build a Bear
-
+# names the function and lists available arguments
```

Comment:
Yep! But in Ruby, we'll use `method` rather than `function`

Reply:
Got it! I'll keep it in mind